english/yell-at-snape/yell-at-snape.md

```diff
@@ -1,5 +1,7 @@
-Hermione, her distaste for Snape's rule-breaking 
-overwhelming her respect for Hogwart's professors 
-screams "How could you do this? Being a Death Eater is 
-one thing but being drunk at Hogwarts is 
-unforgiveable!"
\ No newline at end of file
+Hermione, her distaste for Snape's rule-breaking
+overwhelming her respect for Hogwart's professors,
+screams "How could you do this? Being a Death Eater is
+one thing but being drunk at Hogwarts is
+unforgivable!"
+
+And now on with [Harry's adventure](../magicaladventure/decision_time/decision_time.md) who now faces Voldemort in a duel!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.